definitions/collections/Enemy.py

```diff
@@ -3,14 +3,15 @@
 from definitions.enemy.BossDetails import BossDetails
 from definitions.enemy.EnemyDetails import EnemyDetails
 from definitions.enemy.EnemyNav import EnemyNav
-from definitions.enemy.EnemyTable import EnemyTable
 from definitions.enemy.MapData import MapData
 from definitions.master.IdleonModel import IdleonModel
+from typing import List
+from definitions.common.DropTypes import Drop
 
 
 class Enemy(IdleonModel):
 	details: EnemyDetails
-	drops: Optional[EnemyTable] = None
```

Comment:
I prefer it like this, keeping everything seperate. Enemy table needs to know how to write an enemy table so thats why it is like this

Reply:
Hmm I see, so I can revert that. Is there something we can do to avoid the output being:
```json
"drops": {
   "drops": {
        ...<actual data>
   }
}
```